components/table/doc/index.en-US.md

```diff
@@ -127,6 +127,17 @@ Filter property
 | `(nzFilterChange)` | Filter change callback `value` | `EventEmitter<any[] \| any>` | - |
 
 
+Sort property
+
+| Property | Description | Type | Default |
+| -------- | ----------- | ---- | ------- |
+| `[nzShowSort]` | Whether to show sort | `boolean` | - |
+| `[nzSort]` | Sort direction | `'ascend' | 'descend' | null` | - |
+| `[nzSortKey]` | Key to sort by | `string` | - |
+| `(nzSortChange)` | Sort direction change callback | `EventEmitter<'ascend' \| 'descend' \| null>` | - |
+| `(nzSortChangeWithKey)` | Sort direction change callback with `key` | `EventEmitter<{key: string, value: 'ascend' \| 'descend' \| null}>` | - |
```

Comment:
```suggestion
```

@vthinkxie Looks like a private API, need to make it public in the docs?

Reply:
What makes the `nzSortChangeWithKey` a private api and not the others with a similar signature?

```typescript
  @Output() readonly nzCheckedChange = new EventEmitter<boolean>();
  @Output() readonly nzSortChange = new EventEmitter<string | null>();
  @Output() readonly nzSortChangeWithKey = new EventEmitter<{ key: string; value: string | null }>();
  /* tslint:disable-next-line:no-any */
  @Output() readonly nzFilterChange = new EventEmitter<any[] | any>();
```

Reply:
🤔We should rename it to `sortChangeWithKey`.